packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -210,7 +203,7 @@ contract AnchorStateRegistry is Initializable, ISemver {
         }
 
         // Game must be beyond the airgap period.
```

Comment:
This is now the only use of the term 'airgap' in the contract. I'd suggest removing it and being more specific about the semantic meaning of this check.